features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/default/js/logic/create.asset.js

```diff
@@ -100,7 +99,7 @@ $(function() {
       // data : {"tags":tags},
         success:       function(response) {
 
-				var result = JSON.parse(response);
+				var result = response;
```

Comment:
removed redundant variable 'result' in commit b14b4f54ea23c82a9dad8249923a9e048090eac4 of PR #240